components/static-pages/hiring.md

```diff
@@ -1,47 +1,51 @@
 # Hiring
 
+## We’re hiring!
+
 <!--
-We’re hiring!
+<strong>All position are currently filled, check back later! You can still send a spontaneous application if you wish.</strong>
 -->
 
-<strong>All position are currently filled, check back later! You can still send a spontaneous application if you wish.</strong>
+<em>September 28th 2021</em>. We have an immediate need and are for a looking full stack <strong>free lance</strong> engineer who’s excited about open source, making communities sustainable, and transparent finances. The mission would be initially on a <strong>3 months</strong> contract. This could be later extended based on fit and availability.
 
-<!--
-Seeking a full stack engineer who’s excited about open source, making communities sustainable, and transparent finances.
--->
+## First, about us
 
 Open Collective is on a mission to help communities around the world be sustainable. On Open Collective, groups like open source projects, meetups, and movements can collect and spend money transparently, invite collaboration, and make their communities thrive.
 
 Watch this 2 minute explainer video about what we do and why we are passionate about it.
 
 <iframe src="https://www.youtube.com/embed/IBU5fSILAe8?wmode=opaque&amp;enablejsapi=1" scrolling="no" width="854" height="480" frameborder="0"></iframe>
 
-<!--
-### We think you’d be great at this if you:
+## We think you’d be great at this if you:
+
+- Are comfortable in a remote environment that is at the same time highly collaborative (we [apply](https://github.com/opencollective/opencollective/issues/2267) a [seeking feedback process](http://www.reinventingorganizationswiki.com/Decision_Making)) and autonomous (you can run a feature end to end on your own!)
```

Comment:
@znarf : The links should be https://reinventingorganizationswiki.com/theory/decision-making/ I think (current one is broken). I've opened new PR; https://github.com/opencollective/opencollective-frontend/pull/6942